src/domains/transactions/components/SendModal/SendModal.tsx

```diff
@@ -132,6 +132,11 @@ export const SendModal = ({
             min="0"
             placeholder="Enter Amount"
             step="0.00000001"
+            onInput={(event: FormEvent<HTMLInputElement>) => {
+              event.currentTarget.value = Number(
+                event.currentTarget.value,
+              ).toFixed(8);
```

Comment:
this doesn't work with locales where the decimal separator is `,`, it effectively locks you out of using the input properly . You can reproduce it by setting location in `devtools > sensors` to a european country like NL or DE and try to use the input.